### DISCUSSION TITLE
Ability to target filter button sizes

### DISCUSSION THREAD

**Discussion by ryjogo** [KevinVandy/material-react-table#255] [Ideas]
![image](https://user-images.githubusercontent.com/5095277/207665398-154b9c9a-73f9-4586-8406-857cb5bb8814.png)
It could be nice to be able to scale down the sizes of the filter buttons

**Comment by KevinVandy** [comment]
You can do this in the `muiTableHeadCellFilterTextFieldProps` column option or prop. You can customize any of the props to the text fileld. Or you can target any CSS className using the `sx` prop using the emotion styled components syntax. You'll just have to inspect element to find the classes you want to target.

Something like this is probably what you are looking for

```js
muiTableHeadCellFilterTextFieldProps={{
  sx: {
    '& .MuiInputAdornment-positionEnd': {
      transform: 'scale(0.75)',
      //more of your custom CSS here
    }
  }
}}
```

Learn more in the docs https://www.material-react-table.com/docs/guides/customize-components

**Reply by ryjogo** [reply]
Aha.. now i understand how this works - i think i read that page a few times, thanks!
